mon-pix/tests/integration/components/routes/campaigns/assessment/skill-review-test.js

```diff
@@ -155,4 +159,114 @@ describe('Integration | Component | routes/campaigns/assessment/skill-review', f
     });
 
   });
+
+  describe('The block of the organisation message', function() {
+    context('when the organization has a message', function() {
+      context('when the organization has a logo', function() {
+        beforeEach(async function() {
+          // Given
+          const campaignParticipation = {
+            campaignParticipation: {
+              campaignParticipationResult: {
+                get: sinon.stub().returns([]),
+              },
+              campaign: {
+                customResultPageText: 'Bravo !',
+                organizationLogoUrl: 'www.logo-example.com',
+                organizationName: 'Dragon & Co',
+                get: sinon.stub(),
+              },
+            },
+          };
+          campaignParticipation.campaignParticipation.campaign.get.withArgs('customResultPageText').returns(['Bravo !']);
+          campaignParticipation.campaignParticipation.campaign.get.withArgs('organizationLogoUrl').returns(['www.logo-example.com']);
+          campaignParticipation.campaignParticipation.campaign.get.withArgs('organizationName').returns(['Dragon & Co']);
+          this.set('campaignParticipation', campaignParticipation);
+          // When
+          await render(hbs`<Routes::Campaigns::Assessment::SkillReview
+              @model={{campaignParticipation}}/>`);
+        });
+        it('should display the block for the message', function() {
+          // Then
+          expect(find('.skill-review__organization-message')).to.exist;
+          expect(contains('Message de votre organisation')).to.exist;
+          expect(contains('Dragon & Co')).to.exist;
+        });
+
+        it('should show the logo of the organization ', function() {
+          // Then
+          expect(find('.skill-review-organization-message__logo')).to.exist;
```

Comment:
**suggestion** : cet expect est trop dépendant de la classe, si la classe change, le test devra changer, ce qui est pas top. Pour contrer ça, on peut écrire `expect(find('[src="www.logo-example.com"')).to.exist;`